CodeEdit/Utils/WorkspaceClient/Model/FileIcon.swift

```diff
@@ -44,54 +117,105 @@ enum FileIcon {
         case mod
         case Makefile
         case ts
+        case warningRemover // Removes warning
     }
 
+    // swiftlint:disable function_body_length
     /// Returns a string describing a SFSymbol for files
     /// If not specified otherwise this will return `"doc"`
     static func fileIcon(fileType: FileType) -> String {
         switch fileType {
-        case .json, .js:
+        case .doc, .docm, .docx:
+            return "doc.text"
+        case .lz:
+            return "arrowtriangle.right.fill"
+        case .groovy, .gvy, .gy, .gsh:
+            return "star"
+        case .pkg:
+            return "archivebox"
+        case .dmg:
+            return "externaldrive"
+        case .obj, .dae, .abc, .glb, .stl, .ply, .blend, .gcode:
+            return "move.3d"
+        case .mp3, .wav:
+            return "waveform"
+        case .mov, .mp4:
+            return "video.square"
+        case .app:
+            return "app.dashed"
+        case .asm, .s, .inc, .wla, .src:
+            return "doc.text"
+        case .zip, .tar, .gz, .gzip:
+            return "doc.zipper"
+        case .json, .js, .xml:
             return "curlybraces"
+        case .cobol:
+            return "c.square.fill"
+        case .cs:
+            return "number.square"
         case .css:
             return "number"
         case .jsx:
             return "atom"
+        case .lua:
+            return "l.circle"
+        case .julia:
+            return "j.square"
         case .swift:
             return "swift"
+        case .sql, .scala:
+            return "cylinder.split.1x2"
         case .env, .example:
             return "gearshape.fill"
         case .gitignore:
             return "arrow.triangle.branch"
-        case .png, .jpg, .jpeg, .ico:
+        // swiftlint:disable line_length
+        case .png, .jpg, .jpeg, .ico, .heic, .gif, .tga, .bmp, .psd, .hdr, .pic, .webp, .avif, .ppm, .pgm, .pbm, .pnm, .tiff, .pdf, .ai, .jng, .pns:
             return "photo"
         case .svg:
             return "square.fill.on.circle.fill"
         case .entitlements:
             return "checkmark.seal"
         case .plist:
             return "tablecells"
+        case .php:
+            return "oval.fill"
         case .md, .txt, .rtf:
             return "doc.plaintext"
-        case .html, .py, .sh:
+        case .html, .py, .sh, .htm, .pyc, .pyw:
             return "chevron.left.forwardslash.chevron.right"
+        case .kotlin:
+            return "chevron.left.to.line"
```

Comment:
Using `k` or any `.circle` or `.square` variant is more fitting than a _skip backwards_ symbol